wallet/src/de/schildbach/wallet/ui/coinbase/CoinbaseActivityViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package de.schildbach.wallet.ui.coinbase
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.dash.wallet.integration.coinbase_integration.repository.CoinBaseRepository
+import javax.inject.Inject
+
+@HiltViewModel
+class CoinbaseActivityViewModel @Inject constructor(
```

Comment:
Not sure why the viewModel is also in the wallet module? Should be in coinbase-integration so that we're able to communicate between integration module and the wallet.